sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -507,12 +507,17 @@ public boolean matches(Object item) {
 
     @Test
     public void createTransmissionTargetBeforeStart() {
+
+        /* Given app center not configured. */
         mockStatic(AppCenterLog.class);
+        when(AppCenter.isConfigured()).thenReturn(false);
+
+        /* When creating a target, it returns null. */
         assertNull(Analytics.getTransmissionTarget("t1"));
 
-        /* Verify log. */
+        /* And prints an error. */
         verifyStatic();
-        AppCenterLog.error(anyString(), contains("App context is null, App Center has not been started."));
+        AppCenterLog.error(anyString(), contains("AppCenter is not configured"));
```

Comment:
Add period at end of log message

Reply:
its not the log but the test verification with contains a substring

Reply:
Whoops, should pay closer attention to what I'm looking at 😅